lib/plugins/aws/deploy/lib/createStack.test.js

```diff
@@ -147,6 +147,7 @@ describe('createStack', () => {
 
       return awsDeploy.createStack().then(() => {
         expect(disableTransferAccelerationStub.called).to.be.equal(true);
+        sandbox.restore();
```

Comment:
Restore of the sandbox should be done in `afterEach()` instead of the success path. On failure you will miss the cleanup in your current implementation.

Reply:
Yes sorry, I've reverted that in the next commit :)